mitxpro/templates/terms_of_services.html

```diff
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
+{% load render_bundle %}
+{% load make_fragment_id %}
+{% load wagtailcore_tags %}
+
+{% block headercontent %}
+<div id="header"></div>
+{% render_bundle 'header' %}
+{% endblock %}
+
+{% block content %}
+
+<div class="container tos-container">
+  <div class="row tos-banner">
+    <h1 class="text-uppercase">Privacy Statement</h1>
```

Comment:
This is a little confusing. I'm not sure, could you please confirm if "Privacy Statement" is supposed to appear on this page? It's a ToS page, no?

Reply:
I followed blindly https://impactbnd.invisionapp.com/share/4TQEESTEYMZ#/screens/360916361_-mockup-General_Page, but I guess you are right. 

@pdpinch could you please confirm ?